src/main/resources/db/changelog/changes/10-create-shopping_carts-table.yaml

```
@@ -0,0 +1,26 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-shopping_carts-table
+      author: coder
+      changes:
+        - createTable:
+            tableName: shopping_carts
+            columns:
+              - column:
+                  name: id
+                  type: BIGINT
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: user_id
+                  type: BIGINT
+                  constraints:
+                    nullable: false
```

Primary key will be also a foreign key, because shopping_cart record can't exist without user
```suggestion
              - column:
                  name: user_id
                  type: BIGINT
                  constraints:
                    primaryKey: true
                    nullable: false
```